frontend/src/pages/destructionlist/detail/components/DestructionListReviewer/DestructionListReviewer.tsx

```diff
@@ -67,22 +67,27 @@ export function DestructionListReviewer({
             labeledObject={{
               reviewer: {
                 label: "Beoordelaar",
-                value: formatUser(reviewer.user),
+                value: (
+                  <>
+                    {formatUser(reviewer.user)}
+                    &nbsp;
+                    <Button
+                      aria-label="Beoordelaar bewerken"
+                      disabled={state === "loading" || state === "submitting"}
+                      size="xs"
+                      variant="secondary"
+                      onClick={(e) => {
+                        e.preventDefault();
+                        setModalState(true);
+                      }}
+                    >
+                      <Solid.PencilIcon />
+                    </Button>
```

Comment:
The shape of the button is maybe a bit off?
![Screenshot from 2024-10-15 09-57-20](https://github.com/user-attachments/assets/ec2de4b0-0afd-47a0-bf9c-f9fefa3f9efe)

Reply:
Correct, will fix this later in a admin ui update.